frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.unit.spec.js

```diff
@@ -427,4 +459,42 @@ describe("FieldValuesWidget", () => {
       expect(fetchFieldValues).not.toHaveBeenCalledWith(EXPRESSION_FIELD_ID);
     });
   });
+
+  describe("NoMatchState", () => {
+    it("should display field title when no matching results found", async () => {
+      const metadata = getMetadataStateWithPasswordField({
+        has_field_values: "search",
+      });
+      const fieldId = PEOPLE.PASSWORD;
+      const field = metadata.field(fieldId);
+      const displayName = field.display_name; // "Password"
+      const searchValue = "somerandomvalue";
+
+      fetchMock.get(
```

Comment:
Please move this helper to `server-mocks` like others. There is already a helper for fetching field values. Furthermore, all setup logic should be in `setup`.